gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/jupiter/handlers/api/v4/AsyncApiReactor.java

```diff
@@ -368,9 +369,9 @@ protected void doStop() throws Exception {
     }
 
     private Completable stopUntil() {
-        return interval(100, TimeUnit.MILLISECONDS)
+        return interval(STOP_CHECK_PERIOD_MS, TimeUnit.MILLISECONDS)
```

Comment:
the constant name is not explicit on its use. 
```suggestion
        return interval(STOP_UNTIL_INTERNVAL_PERIOD_MS, TimeUnit.MILLISECONDS)
```